specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,616 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview data policies REST API Document",
+    "version": "2023-08-01-preview"
+  },
+  "paths": {
+    "/data-policies": {
+      "get": {
+        "tags": [
+          "List_DataPolicy"
+        ],
+        "summary": "API to list the purview data policies",
+        "description": "List all data policies.",
+        "operationId": "List_DataPolicy",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicyList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy service",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
```

Comment:
Don't cross the streams. Don't reference resource-management common types. Define your own error model either in this file or in a Purview (data-plane) common types file.

Reply:
One can actually use the shared data-plane types https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/data-plane/v1/types.json

Reply:
Used shared data-plane types as suggested.